packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -32,11 +32,11 @@ const { VaultPriceFeed } = require("./VaultPriceFeed");
 const uniswapBlockCache = {};
 
 async function createPriceFeed(logger, web3, networker, getTime, config) {
-  const Uniswap = getTruffleContract("Uniswap", web3, "latest");
-  const ERC20 = getTruffleContract("ExpandedERC20", web3, "latest");
-  const Balancer = getTruffleContract("Balancer", web3, "latest");
-  const VaultInterface = getTruffleContract("VaultInterface", web3, "latest");
-  const Perpetual = getTruffleContract("Perpetual", web3, "latest");
+  const Uniswap = getTruffleContract("Uniswap", web3);
+  const ERC20 = getTruffleContract("ExpandedERC20", web3);
+  const Balancer = getTruffleContract("Balancer", web3);
+  const VaultInterface = getTruffleContract("VaultInterface", web3);
+  const Perpetual = getTruffleContract("Perpetual", web3);
```

Comment:
Should we pin the perp here? Or should we use latest?

More generally, how did you decide where to pin vs use latest for contracts that _we've_ launched (the interfaces above should obviously use latest)?

Outside of this particular instance, it looks like we're pinning in the src, but using latest in the tests. Is that correct? That seems reasonable, just want to make sure I'm understanding correctly.

Reply:
we probably should pin the perp version at this point but I actually don't think it will matter as the downstream implementations are expecting any `Perpetual` that implements the right interface. As long as we don't shift the interface it wont break anything to keep it like this (or how the code was before).

My main decision was if things used `latest` within the `getTruffleContract` then the param is redundant. If `latest` was used within unit tests (in particular within a `versionedIt`) then it should be a locked version and I used the `2.0.1` release bytecode.